tailwind.config.cjs

```diff
@@ -1,8 +1,25 @@
+const primaryColor = '#872020FF';
+const secondaryColor = '#B33636FF';
+const tertiaryColor = '#C04343FF';
+const tertiary60Color = '#C0434399';
```

Comment:
Any reason for not putting them directly in the config? I'm not against it, just asking

Reply:
In case the colors are needed elsewhere further in the development, they'll be isolated from the tailwind config and are easier to change globally later on. If they end up not being used outside of tailwind colors, then we can add them directly in the config.